packages/wdio-config/src/shim.js

```diff
@@ -25,7 +25,7 @@ export let executeHooksWithArgs = async function executeHooksWithArgsShim (hooks
             result = hook.apply(null, args)
         } catch (e) {
             log.error(e.stack)
-            return resolve()
+            return resolve(e)
```

Comment:
Are we using this error somewhere? I am not sure if it is a good idea to pass the error as result.

Reply:
Yes, we use it, I think we have just missed this behavior in shim.
It is used in logHookError